ui/StatusQ/src/stringutilsinternal.cpp

```diff
@@ -32,27 +33,24 @@ QString StringUtilsInternal::escapeHtml(const QString& unsafe) const
     return unsafe.toHtmlEscaped();
 }
 
-QString StringUtilsInternal::readTextFile(const QString& filePath) const
+QString StringUtilsInternal::readTextFile(const QString& filePath)
```

Comment:
Why this method is made non-const? :thinking:

Reply:
Because of `auto engine = qmlEngine(this);`

Reply:
how is it related?